_sass/layout/_portfolio.scss

```diff
@@ -0,0 +1,140 @@
+// Styling for the portfolio section
```

Comment:
Do we still need this file? Not sure if your simplification took out the portfolio section altogether...

Reply:
or would you recommend keeping in case we decide to use it in the future?

Reply:
> Do we still need this file? Not sure if your simplification took out the portfolio section altogether...

Not need to keep it